misc-tools/dj_make_chroot.in

```diff
@@ -37,19 +37,22 @@ UBUNTU_ARCHLIST="amd64,arm64,armhf,i386,powerpc,ppc64el,s390x"
 # If host system is Debian or Ubuntu, use its release and architecture by default
 if [ "$(lsb_release -i -s || true)" = 'Debian' ] || \
    [ "$(lsb_release -i -s || true)" = 'Ubuntu' ]; then
-    DISTRO="$(lsb_release -i -s)"
-    RELEASE="$(lsb_release -c -s)"
-    if [ "$(lsb_release -i -s)" = 'Debian' ]; then
-        if [ "$(lsb_release -c -s)" = "sid" ]; then
-            RELEASE='unstable'
-        fi
-        if [ "$(lsb_release -r -s)" = 'testing' ]; then
-            RELEASE='testing'
-        fi
-    fi
     if [ -z "$ARCH" ]; then
         ARCH="$(dpkg --print-architecture)"
     fi
+    # If Distro & Release are not specified pick the host value
+    if [ -z "$DISTRO" ] && [ -z "$RELEASE" ]; then
```

Comment:
This won't set the release if only `DISTRO` is set, as is the case above.

Reply:
You're right, by looking at the code we still would have an unwanted situation:

Host: Debian/oldstable
dj_make_chroot
Host detection uses all host values
Chroot: Debian/oldstable 

Host: Debian/oldstable
dj_make_chroot -D Debian
We use the Distro=Debian && -z RELEASE
Chroot: Debian/stable

I think we can keep that one as it would make the code a lot harder otherwise.